include/library/spdm_common_lib.h

```diff
@@ -121,6 +121,16 @@ typedef enum {
      **/
     LIBSPDM_DATA_REQUEST_RETRY_DELAY_TIME,
 
+    /* Below two entries are used to limite the number of DHE session and PSK session separately.
+     * When set a new value, below rule is applied:
```

Comment:
This should also state that if these values are modified while there are active sessions then the active sessions aren't terminated. For example if there are two PSK sessions and `LIBSPDM_DATA_MAX_PSK_SESSION_COUNT` is changed to `1` then one of the sessions won't be terminated.

Reply:
done